### PR TITLE
ccl-2022-ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2022.ccl.xml [hunk 1]
```diff
@@ -0,0 +1,1316 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.ccl">
+  <volume id="1" ingest-date="2022-09-19">
+    <meta>
+      <booktitle>"Proceedings of the 20th Chinese National Conference on Computational Linguistics"</booktitle>
```
Comment (mjpost) [inline data/xml/2022.ccl.xml:5]: no quotes here

File: data/xml/2022.ccl.xml [hunk 2]
```diff
@@ -0,0 +1,1316 @@
...
+      <editor><first>Gaoqi</first><last>Rao</last></editor>
+      <editor><first>Yubo</first><last>Chen</last></editor>
+      <editor><first>茂松</first><last>孙</last></editor>
+      <editor><first>洋</first><last>刘</last></editor>
+      <editor><first>万翔</first><last>车</last></editor>
+      <editor><first>洋</first><last>冯</last></editor>
+      <editor><first>锡鹏</first><last>邱</last></editor>
+      <editor><first>高琦</first><last>饶</last></editor>
+      <editor><first>玉博</first><last>陈</last></editor>
+      <publisher>Chinese Information Processing Society of China</publisher>
```
Comment (mjpost) [inline data/xml/2022.ccl.xml:20]: Is this everyone's name once in Chinese, once in English? That's not right...

Comment (mjpost) [reply data/xml/2022.ccl.xml:20]: This should all be done as e.g.,

`<editor><first>Maosong</first><last>Sun</last><variant><first>茂松</first><last>孙</last></variant></editor>`

Same for all the authors below.

File: data/xml/2022.ccl.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2022.ccl.xml:5]: Also, this is the 21st, not the 20th

**Comment by xinru1414** [comment]
waiting for an update on the ingestion material

**Review by mjpost — APPROVED** [review]

**Pull request merged**